content/doc/book/managing/system-configuration.adoc

```diff
@@ -13,3 +13,114 @@ endif::[]
 
 = Configuring the System
 
+Jenkins can be configured at many levels and with many different techniques.
+Your final configuration depends directly on which problems you want to solve.
+
+Jenkins has a specific area all configurations items can be accessed.
+Configuration layers might include Jenkins master configuration, 
```

Comment:
FTR we will be renaming the "Jenkins master" term soon: https://groups.google.com/forum/#!topic/jenkinsci-dev/CLR55wMZwZ8. Fine to keep it here for now, will require update later.